src/modules/72-templates-library/components/TemplateStageSpecifications/TemplateStageSpecifications.tsx

```diff
@@ -202,7 +202,7 @@ export const TemplateStageSpecifications = (): JSX.Element => {
         >
           {(formik: FormikProps<TemplateStageValues>) => {
             window.dispatchEvent(new CustomEvent('UPDATE_ERRORS_STRIP', { detail: TemplateTabs.OVERVIEW }))
-            formikRef.current = formik
+            formikRef.current = formik as any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
            formikRef.current = formik as unknown
```